SwiftWisdom/Core/Foundation/Data/Data+Extensions.swift

```diff
@@ -99,11 +99,6 @@ extension Data {
         return self[ip_safely: idx...idx]
     }
 
-    /// Safer version of `subdata`. Will return nil if the range is outside the bounds of the data.
-    public subscript(ip_safely range: CountableRange<Int>) -> Data? {
```

Comment:
Was `CountableRange` just removed in 4.2?

I'm not finding any information around that online.

Reply:
See https://github.com/apple/swift/blob/master/stdlib/public/core/Range.swift#L287

Reply:
This would still be needed for Swift < 4.2 though, no?

Reply:
The problem is that I can't seem to find a good way to only build this code if you're NOT on Swift 4.2

Reply:
Would this work?
```swift
#if swift(>=4.2)
#else
    /// Safer version of `subdata`. Will return nil if the range is outside the bounds of the data.
    public subscript(ip_safely range: CountableRange<Int>) -> Data? {
        return self[ip_safely: Range<Int>(range)]
    }
#endif
```

Reply:
@bobgilmore nope, because the compiler version in Xcode 10 still flags it as an invalid redeclaration.

Until this proposal is implemented in one of the 4.2 releases I think we should hold off on this PR.
https://github.com/apple/swift-evolution/blob/master/proposals/0212-compiler-version-directive.md